README.md

```diff
@@ -18,10 +18,11 @@ And then you need to configure a mapping to visually select accordingly to your
 local nvim_lsp = require('lspconfig')
 local lsp_selection_range = require('lsp-selection-range')
 
--- Generate the default Neovim LSP capabilities
-local capabilities = vim.lsp.protocol.make_client_capabilities()
--- Update the default capabilities
-capabilities = lps_selection_range.update_capabilities(capabilities)
+-- Create the capabilities
+local capabilities = lsp_selection_range.default_capabilities()
+
+-- Or update existing capabilities
+-- capabilities = lsp_selection_range.update_capabilities(capabilities)
```

Comment:
```suggestion
-- Update existing capabilities
capabilities = lsp_selection_range.update_capabilities(capabilities)

-- If you don't already have custom capabilities you can simply do
local capabilities = lsp_selection_range.update_capabilities({})
```

I don't really like the idea of "creating LSP capabilities" from such a plugin.
It's not really it's job, it should only work around capabilities provided by the user.
Which means the `update_capabilities` method has to be updated to stop using the `vim.lsp.protocol.make_client_capabilities()` as well.

Reply:
Sounds reasonable. I guess `default_capabilities()` could be misleading in that it sounds like it is providing a complete set of capabilities.